### DISCUSSION TITLE
OK but Malloc Error

### DISCUSSION THREAD

**Discussion by MAWemah** [Tripouille/gnlTester#13] [Q&A] [answered]
Hello, I'm another 42 student writing a get_next_line, and while my own tests work well, when I use the tester I'm receiving this error along with my "OK":

> gnlTest(33325,0x1120d9dc0) malloc: *** error for object 0x10967b250: pointer being freed was not allocated
> gnlTest(33325,0x1120d9dc0) malloc: *** set a breakpoint in malloc_error_break to debug

I have been fiddling around in both my code and the tester to understand what is causing the error but couldn't find it, any ideas?

**Comment by Tripouille** [comment, marked as answer]
Hi, looks like you're trying to free a not allocated pointer. Maybe you call free on an uninitialized pointer?

**Reply by MAWemah** [reply]
Yeah it was, thank you
